deploy/setup.sh

```
@@ -0,0 +1,322 @@
+
+bold "1. cosign"
+if command -v cosign >/dev/null; then
+	info "already installed: $(cosign version 2>&1 | head -1)"
+else
+	info "downloading latest release..."
+	tmpfile="$(mktemp)"
+	trap 'rm -f "$tmpfile"' EXIT
+	curl -sSL -o "$tmpfile" \
+		https://github.com/sigstore/cosign/releases/latest/download/cosign-linux-amd64
+
+	curl -sSL -o "${tmpfile}.sha256" \ 
```

Shellcheck (0.11.0)

[error] Delete trailing spaces after \ to break line (or use quotes for literal space).

(SC1101)